plugins/playlist/src/components/PlaylistPage/PlaylistEntitiesTable.tsx

```diff
@@ -73,16 +74,24 @@ export const PlaylistEntitiesTable = ({
   const removeEntity = useCallback(
     async (_, entity: Entity | Entity[]) => {
       try {
+        const entityName = [entity]
+          .flat()
+          .map(item => item.metadata.title ?? item.metadata.name);
         await playlistApi.removePlaylistEntities(
           playlistId,
           [entity].flat().map(stringifyEntityRef),
         );
         loadEntities();
+        alertApi.post({
+          message: `Removed entity '${entityName}'`,
```

Comment:
This may be one or more entity names, thinking we'll want different messages for those cases. Right now you might get something like `Removed entity 'service-a, service-b'`

Reply:
@Rugvip sure pushed the changes please check and let me know if this ok